code/modules/clothing/modular_armor/attachments/chest_plates.dm

```diff
@@ -95,3 +95,24 @@
 	soft_armor = list(MELEE = 15, BULLET = 35, LASER = 35, ENERGY = 25, BOMB = 25, BIO = 25, FIRE = 25, ACID = 25)
 	slowdown = 0.1
 	greyscale_config = /datum/greyscale_config/hardsuit_variant
+
+//VSD Hardsuits
+/obj/item/armor_module/armor/chest/marine/vsd_hardsuit
+	name = "\improper Crasher Super-Heavy MT/41 'Phobos' chestplate"
+	desc = "Designed for use with the CrashCore MT/P Exoskeleton. It provides a very good amount protection, with the cost of encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. Meaning that only changing the chestplate makes significant armor difference. This armor appears to be marked as a 'Phobos' armor piece."
+	soft_armor = list(MELEE = 70, BULLET = 70, LASER = 70, ENERGY = 50, BOMB = 45, BIO = 45, FIRE = 45, ACID = 50)
```

Comment:
Never ever do this again, especially without mentioning it in the PR description with bold letters. We've clearly established we do not want marines to be tanky, no amount of speed penalty would ever be enough to make this amount of armor okay. 

Sneaking stuff like this in is really scummy and won't be tolerated.